src/partials/projects/projects.html

```diff
@@ -1 +1,166 @@
-<section class="projects">PROJECTS</section>
+<section class="projects">
+    <div class="swiper-container container">
+        <div class="projects-header">
+            <h2>PROJECTS</h2>
+          </div>
+      <div class="swiper-wrapper">
+        
+        <div class="swiper-slide">
+          <ul class="project-list">
+
+
+            <li class="project-list-item">
+                <div class="project-card">
+                  <div class="left-part">
+                    <div class="technologies">
```

Comment:
In my opinion, if there is a list of something, it is better to avoid <div> and use <ul><li>...</li></ul>.
```html
                <ul class="technologies">
                  <li class="technology">#react</li>
                  <li class="technology">#js</li>
                  <li class="technology">#node js</li>
                  <li class="technology">#git</li>
                </ul>
```